### ISSUE TITLE
Gliner finetuning fail on the notebook example

### ISSUE THREAD

**Issue by yanisdaci** [urchade/GLiNER#212]
Hello everyone,

I am trying to fine-tune the "knowledgator/gliner-llama-1.3B-v1.0" model using the finetune notebook example but validation loss is always "nan" and after the first step my Training loss is constantly equal to 0 
Does someone know what could be the issue ?
Thanks !
![Screenshot from 2024-11-25 11-46-37](https://github.com/user-attachments/assets/0c613cd2-0a0f-4f69-b119-39d63e314c99)
